x-pack/test/functional/apps/observability_log_explorer/columns_selection.ts

```diff
@@ -30,6 +30,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
     describe('when the log explorer loads', () => {
       it("should initialize the table columns to logs' default selection", async () => {
+        // did this work?
+        PageObjects.settings.refreshDataViewFieldList('All logs');
```

Comment:
This test fails because the "All logs" is an ad-hoc data view.
As you can see in the screenshot from [this job](https://buildkiteartifacts.com/e0f3970e-3a75-4621-919f-e6c773e2bb12/0fda5127-f57f-42fb-8e5a-146b3d535916/018bb7e1-5720-49e6-bfbe-7bdd082ca234/018bb811-55c5-4c05-9b38-28aba280bebe/target/test_failures/018bb811-55c5-4c05-9b38-28aba280bebe_ae8a51a7097d1ad9cc6f5c332f440623.html?response-content-type=text%2Fhtml&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAQPCP3C7LYTCU2R5I%2F20231110%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20231110T130900Z&X-Amz-Expires=600&X-Amz-Security-Token=IQoJb3JpZ2luX2VjELH%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJGMEQCIDWDLfDQIX6ludQAESY0WrttIrqvfQwKpk56iL9iwN4GAiAcEYC7LV5bmzEv%2FHxNdtfMoCVk24p75QBB0V1rI%2FFzZir6Awjp%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F8BEAAaDDAzMjM3OTcwNTMwMyIMpvnfRpf1wnBNaVi2Ks4Dh8JYsZvokcDiNeCa70NZQb77hrm8xDfz0ZKy0wmAlCfG7OKSfq3rWXMjGrILHiVA0Cbiard%2B7tKILc2y4NE82qigGVmVw5pnhyhmAGuOyz4QiA%2BV60nbIRwxzq87DeWuGRITfnZM%2BtQiIeeAJ9L%2Fi3Jx6OL1%2F7QiJl4L%2Bm%2BVlQ%2FGKNQMDDcjqFrv5lyEzHwtzJPj6DtAPq%2BGRJ3KxkcL9kkSgN2AbArHrvmTXmGq9zg%2FSkxqzqs%2F%2B6bk6qoJzo99JjCY0xz5LndSwdeCjNEKllfiwYzDr2f3jfAI7l%2BDYsB5oI9Sdd9RRvTrFPXiIIwPgt9wXihi7qBJHrbVUSqHgmXQ4Q50BpOLIV66S9Q0NO8OPMrgpU88YiDB8YXjRv7XLQWi2Wr3%2BxFq9dv%2FOGclA7FE7Ff%2BUlDy7y8LMx5gkPCpLf21gwQQ0JC7udjJh0vpZGICJcDRIhN3ijYUDYkjS3P1LLIhxmBaT%2BXrCtVdC%2BV9gzOiLhUfepozREOO2PUaMur680e5gNCtoqdbU6hpFqapd7ZIYmjDy9Wl8hS8azQnU9iOz60VEfrMQDvKlDEKwgqNnO3ju8gu83neAenh%2BQ8vORh2vXSwxp20oVTGMJjQt6oGOqYBGik1t0xg2rimHi093dVZnqrdw6xgXsMevW2p7m3ALV2Fof3204CPPM4nexSy%2FOSFrJt%2FqldssR18s2yznrekJhXZTzgreq5LYQkHNf2OxfAtpK%2BkpM3B6mcYL5Wd%2BEMCDNLgXWf2zcY%2BEkSRPPEhxPOt%2F3BLvNHKHerpPkoNZPbP%2FQPJacOZMslryeglo7UeBUfci5Y%2B9oeV0oSluqDqOgY0pBNwzA%3D%3D&X-Amz-SignedHeaders=host&X-Amz-Signature=4424a2d4eb5d0b539c03db7acc2197f2c23aa16b4e9a4cfa977780217325e826), when you land on the `Stack Management -> Data views` and try to access the "All logs" data view, it gets stuck on the empty prompt, causing the test to fail later for missing the missing data on the page.

I wonder if it makes sense to test it here since this feature is embedding the Discover container with all its logic, which I see has already been tested for your changes.